src-plugins/itkFilters/itkFiltersMedianProcess.cpp

```diff
@@ -43,6 +52,16 @@ QString itkFiltersMedianProcess::description() const
 
 //-------------------------------------------------------------------------------------------
 
+void itkFiltersMedianProcess::setParameter(double data, int channel)
```

Comment:
You could use the method setParameter without channel `void setParameter(double data)`

Reply:
Ok I used it to stay consistent with all the other filters